.github/actions/paths-filter/action.yml

```diff
@@ -155,6 +155,13 @@ outputs:
         github.event_name == 'schedule' ||
         steps.filter-common.outputs.openapi-change == 'true'
       }}
+  stable-branch-changes:
+    description: Output whether the change is occurring on a stable branch or not
+    value: >-
+      ${{
+        github.event_name == 'push' &&
+        steps.filter-stable-branch.outputs.stable-branch == 'true'
+      }}
```

Comment:
:thought_balloon: I was kind of expecting this value to be a bool already, so a bit confused why it's a string. Inputs can be booleans, but not outputs? :thinking:

Reply:
Because sadly GHA is messy like that and has an inconsistent type system. Job inputs/outputs got introduced as a concept way before e.g. `workflow_call` inputs so only the latter feature that type system, while the former are always strings without a way to specify a type 😞